.github/workflows/build_and_push_images.yml

```diff
@@ -20,6 +20,11 @@ jobs:
         - 
           name: Check out the repo
           uses: actions/checkout@v3
+        -
+          name: Use go version
+          uses: actions/setup-go@v5
+          with:
+              go-version: '>=1.22.4'
```

Comment:
I would use 

```
        with:
          go-version-file: go.mod
```

instead to keep it up to date whenever we update the go version inside `go.mod` file

Reply:
oh nice

Reply:
pushed

Reply:
Actually we don't need the latest version of Go because the image is built on the base image and not on the daemon environment. What I like here is setup-go@v5 it has certain improvements.

Reply:
ok, merge?